docs/guide/built-in-rules-callback.md

```diff
@@ -87,10 +86,10 @@ new Callback(
 > **Note:** Processing untrusted user input with `yaml_parse()` can be dangerous with certain settings. Please refer to
 > [yaml_parse docs] for more details. 
 
-### Usage of validation context for validating multiple attributes depending on each other
+### Using a validation context to validate interdependent attributes
 
-A validation context can be utilized too - for example, when performing validation of attributes depending on each
-other. In the below example the 3 angles are verified by degree values to form a valid triangle:
+A validation context can also be used. In the example below, the 3 angles are validated as degrees to form
```

Comment:
> when performing validation of attributes depending on each other

Why delete explanation part?

Reply:
It was suggested by Chris beucause it's clear from the header above.

Reply:
This reads incomplete now: "A validation context can also be used.". Maybe delete it at all.